mathext/hyp2f1_scipy_test.go

```diff
@@ -0,0 +1,54 @@
+package mathext
+
+import (
+	"fmt"
+	"math"
+	"testing"
+)
+
+func TestHyp2f1_Miscellaneous(t *testing.T) {
+	t.Parallel()
+
+	// Taken from test_hyp2f1.py test_miscellaneous.
+	y := h(1.3, -0.2, 0.3, -2.1)
+	expected := 1.8202169687521206
+	if d := y - expected; math.Abs(d) > 5e-15 {
+		t.Errorf("%f %f %f", y, expected, d)
+	}
+}
+
+func TestHyp2f1_SomeRealPoints(t *testing.T) {
+	t.Parallel()
+
+	// Taken from test_mpmath.py test_hyp2f1_real_some_points.
```

Comment:
This needs a link and if it's a translation as opposed to just taking a set of constants, this has license implications.